docs/_includes/topbar.html

```diff
@@ -24,6 +24,16 @@
                     <a class="nav-link cw-nav-link cw-header-link-text" href="guides.html">Guides</a>
                 </li>
 
+                <form class="form-inline my-2 my-lg-0" action="{{ site.baseurl | append:  '/' }}search.html" method="get">
```

Comment:
Search bar aligned slightly above other topbar elements, would be good to get design/docs opinion on search-bar in the topbar vs link to search page.